ivy/functional/backends/tensorflow/module.py

```diff
@@ -3,13 +3,35 @@
 import re
 import os
 import tensorflow as tf
+import keras
+import numpy as np
 import functools
 from tensorflow.python.util import nest
-from typing import NamedTuple, Callable, Any, Tuple, List, Dict, Type, Union
+from typing import (
+    NamedTuple,
+    Callable,
+    Any,
+    Tuple,
+    List,
+    Dict,
+    Type,
+    Union,
+    TYPE_CHECKING,
+)
 import inspect
 from collections import OrderedDict
 from packaging.version import parse
-import keras
+
+if TYPE_CHECKING:
+    import torch.nn as nn
+
+
+if parse(keras.__version__).major > 2:
```

Comment:
Just fyi, I think this line can be simplified to this
```suggestion
if keras.__version__ >= "3.0.0":
```

Reply:
I think both are equivalent but yeah this also works 👍🏼